www/main/navigator/shabad/ShabadContent.jsx

```diff
@@ -1,10 +1,14 @@
+import Noty from 'noty';
 import React, { useState, useEffect, useRef } from 'react';
 import { useStoreState, useStoreActions } from 'easy-peasy';
-import { ipcRenderer } from 'electron';
+import { ipcRenderer, remote } from 'electron';
 
+import copy from 'copy-to-clipboard';
```

Comment:
I was wondering do we really need this package? I added this because this was already in the project but without this package, it's possible to use navigator API.

Reply:
@theamanjs Let's do that as a separate PR.